### ISSUE TITLE
#search input is missing required attribute aria-expanded.

### ISSUE THREAD

**Issue by BezV8** [magento/magento2#18337]
Fix from issue #2681 seems to have regressed and a blank install of 2.2.6 is again showing errors on the W3C html validator for the #search element.

### Preconditions

1. Magento 2.2.6
2. Magento Blank theme enabled.

### Steps to reproduce

1. Check homepage in W3C html validator.

### Expected result

1. After issue #2681 closed validation should be clean.

### Actual result

1. Validator is again showing error for #search.

**Comment by magento-engcom-team** [comment]
Hi @BezV8. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@BezV8 do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by BezV8** [comment]
@magento-engcom-team give me 2.2.6 instance

**Comment by magento-engcom-team** [comment]
Hi @BezV8. Thank you for your request. I'm working on Magento 2.2.6 instance for you

**Comment by magento-engcom-team** [comment]
Hi @BezV8, here is your [Magento instance](https://i-18337-2-2-6.engcom.dev.magento.com).
Admin access: [https://i-18337-2-2-6.engcom.dev.magento.com/admin](https://i-18337-2-2-6.engcom.dev.magento.com/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by BezV8** [comment]
@magento-engcom-team Issue confirmed on test instance.

**Comment by BezV8** [comment]
Screenshot of validation error:
![screenshot_2018-10-02 showing results for https i-18337-2-2-6 engcom dev magento com - nu html checker](https://user-images.githubusercontent.com/33320744/46345195-90d44280-c63b-11e8-93fe-33b16271d8cb.png)

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-andrii. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [ ] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [x] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [x] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [x] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [x] 6. Add label `Issue: Confirmed` once verification is complete. 

- [x] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by ghost** [comment]
@magento-engcom-team give me 2.2-develop instance

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-andrii. Thank you for your request. I'm working on Magento 2.2-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-andrii, here is your [Magento instance](https://i-18337-2-2-develop.instances.magento-community.engineering).
Admin access: [https://i-18337-2-2-develop.instances.magento-community.engineering/admin](https://i-18337-2-2-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by magento-engcom-team** [comment]
@engcom-backlog-andrii Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-96698`, `MAGETWO-96699` were created

**Comment by geet07** [comment]
@magento-engcom-team give me 2.3-develop

**Comment by geet07** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @geet07. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @geet07, here is your [Magento instance](https://i-18337-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-18337-2-3-develop.instances.magento-community.engineering/admin](https://i-18337-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by magento-engcom-team** [comment]
Hi @BezV8. Thank you for your report. 
The issue has been fixed in magento/magento2#22942 by @geet07 in 2.3-develop branch
Related commit(s):
 - [eeae4b240ec62d69ed5848afa505c88539ff1314](https://github.com/magento/magento2/commit/eeae4b240ec62d69ed5848afa505c88539ff1314)
 - [c3955b25f4e7372cf5f73b11a9caa53a4fba2511](https://github.com/magento/magento2/commit/c3955b25f4e7372cf5f73b11a9caa53a4fba2511)
 - [bf4fbf029681851104d95695e11aa22022adb69b](https://github.com/magento/magento2/commit/bf4fbf029681851104d95695e11aa22022adb69b)
 - [81c722d0ca304a57acc8b1297fceed7dd8054225](https://github.com/magento/magento2/commit/81c722d0ca304a57acc8b1297fceed7dd8054225)

The fix will be available with the upcoming 2.3.3 release.

**Comment by magento-engcom-team** [comment]
Hi @BezV8. Thank you for your report. 
The issue has been fixed in magento/magento2#23331 by @amol2jcommerce in 2.2-develop branch
Related commit(s):
 - [83a9c9b16a9d00590491a64684de65702e7d7956](https://github.com/magento/magento2/commit/83a9c9b16a9d00590491a64684de65702e7d7956)
 - [39e7c2cbda322b7aac524fca6fa7566436bf143d](https://github.com/magento/magento2/commit/39e7c2cbda322b7aac524fca6fa7566436bf143d)
 - [3d989c6d0bf16b6b38d66bd1fdc9e80b77af5a68](https://github.com/magento/magento2/commit/3d989c6d0bf16b6b38d66bd1fdc9e80b77af5a68)

The fix will be available with the upcoming 2.2.10 release.
